dEdX/include/dEdxAnalyser.h

```diff
@@ -99,8 +133,8 @@ class dEdxAnalyser : public Processor {
   
   virtual void end();
 
-  virtual void PlotTH1( TCanvas* can, TImage* img, TH1* hist );
-  virtual void PlotTH2( TCanvas* can, TImage* img, TH2* hist );
+  virtual void PlotTH1( TCanvas* can, TH1* hist );
```

Comment:
Do these need to be member functions? From the definition in the .cc file it looks like they could also be free-standing functions, defined in the .cc file
```cpp
template<class TH>
void PlotTH(TCanvas* can, TH* hist, const std::string& opt="") {
  hist->Draw(opt.c_str());
  can->Update();
  std::stringstream s; s << hist->GetName() << _fileFormat;
  can->Print(s.str().c_str());
}
```
And then calling them would simply be
```cpp
// PlotTH1(can, hist);
PlotTH(can, hist);

// PlotTH2(can, hist);
PlotTH(can, hist, "colz");
```
Alternatively, if you are not comfortable with the templated version, you could also just use `TH1*` as second argumend, since `TH2` is a derived class from `TH1*`.

Reply:
strictly optional.